src/spec/unit.ts

```diff
@@ -42,7 +42,7 @@ export type NormalizedUnitSpec = GenericUnitSpec<Encoding<Field>, Mark | MarkDef
 /**
  * A unit specification, which can contain either [primitive marks or composite marks](https://vega.github.io/vega-lite/docs/mark.html#types).
  */
-export type UnitSpec = GenericUnitSpec<CompositeEncoding, AnyMark>;
+export type UnitSpec = GenericUnitSpec<CompositeEncoding, AnyMark> & DeprecatedFrameMixins;
```

Comment:
It's a bit funny that UnitSpec and UnitSpecWithFrame are pretty much the same, but one has "deprecated frame mixins".

Reply:
Oh well.